### PR TITLE
Enforce open url files with system actions

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/9968

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-pkg/src/composables/actions/files/useFileActions.ts [hunk 1]
```diff
@@ -276,6 +276,11 @@ export const useFileActions = ({ store }: { store?: Store<any> } = {}) => {
       return []
     }
 
+    // enforce open url files with system actions
+    if (options.resources[0].extension === 'url') {
```
Comment (JanAckermann) [inline packages/web-pkg/src/composables/actions/files/useFileActions.ts:280]: .url files will have 'application/ocet-stream' as mimetype. Which is assigned to Collabora and OnlyOffice, therefore url files will be opened with respective editors. 

We might fix it in the frontend with this code or need to ask OCIS to return another mime-type like  
https://github.com/owncloud/web/issues/9968

**Comment by JanAckermann** [comment]
Alternative backend solution: https://github.com/cs3org/reva/pull/4344

**Review by dschmidt — APPROVED** [review]
I would prefer the backend solution, but if they can't/don't want to do it, I'm fine with this as welll

**Comment by JanAckermann** [comment]
> I would prefer the backend solution, but if they can't/don't want to do it, I'm fine with this as welll

Same here. If ocis/reva PRs are merged, I'll close this.

**Comment by JanAckermann** [comment]
Closed in favor of https://github.com/cs3org/reva/pull/4344

**Pull request closed without merging**